src/main/java/io/openliberty/sample/MyApplication.java

```diff
@@ -9,20 +9,11 @@
  *     IBM Corporation - Initial implementation
  *******************************************************************************/
 
-package io.openliberty.sample.config;
-
-import java.util.HashSet;
-import java.util.Set;
+package io.openliberty.sample;
 
 import javax.ws.rs.ApplicationPath;
 import javax.ws.rs.core.Application;
 
-@ApplicationPath("config")
-public class ConfigApplication extends Application {
-    @Override
-    public Set<Class<?>> getClasses() {
-        Set<Class<?>> resources = new HashSet<>();
-        resources.add(ConfigResource.class);
-        return resources;
-    }
+@ApplicationPath("")
+public class MyApplication extends Application {
 }
```

Comment:
This is a nit, really, but "My" in the application name doesn't quite seem right.  Maybe something like "MPApplication" or "JaxRSApplication" or "SampleApplication".  Probably the latter given the package name is io.openliberty.sample.

Reply:
Yes, that is a much better name. I have updated the PR.

Reply:
Thanks :)